science/multimon-ng/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        EliasOenal multimon-ng 1.1.7
+
+platforms           darwin
+categories          science comms
+license             GPL-2.0
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+
+description         multimon-ng decodes radio digital transmission signals
+long_description    multimon-ng is the successor of multimon. \
+    It decodes the following digital transmission modes: \
+    POCSAG512 POCSAG1200 POCSAG2400 FLEX EAS UFSK1200 CLIPFSK \
+    AFSK1200 AFSK2400 AFSK2400_2 AFSK2400_3 HAPN4800 FSK9600 \
+    DTMF ZVEI1 ZVEI2 ZVEI3 DZVEI PZVEI EEA EIA CCIR MORSE_CW X10
+
+checksums           rmd160  c4fbc0b7bf6557e6669cf5df43b94b6fb794202f \
+                    sha256  91bbe6f39bac840fea3e64e84b9ec7461ba1e74048369532591a31dcce84d94c \
+                    size    2430248
+
+# disable pulseaudio and x11
+patchfiles-append   \
+    patch-cmakelists.txt_pulseaudio.diff \
+    patch-cmakelists.txt_x11.diff
+
+variant pulseaudio description {Enable Pulseaudio support} {
+    patchfiles-delete  \
+        patch-cmakelists.txt_pulseaudio.diff
+
+    depends_lib-append \
+        port:pulseaudio
+}
+
+variant x11gui description {Enable x11 interface} {
```

Comment:
I would call this variant just x11, as thats commonly what variants enabling X11 support are called.

Reply:
I removed `description {}` to avoid `port lint` warning
`Warning: Variant x11 overrides global description`